README.md

```diff
@@ -115,6 +117,31 @@ branch in your fork to the branch it originated from in [ACCESS-NRI/access-om3-c
 
 ## Configuration CI
 
+### Config Branches
```

Comment:
Should we add this section later (i.e. when we do our first release ?)

Reply:
Removed in https://github.com/ACCESS-NRI/access-om3-configs/pull/122/commits/db99e671d499d819b3423db87b30156bb675453b